lib/dentaku/ast/functions/ruby_math.rb

```diff
@@ -1,8 +1,53 @@
 # import all functions from Ruby's Math module
 require_relative '../function'
 
+module Dentaku
+  module AST
+    class RubyMath < Function
+      def self.[](method)
+        klass = Class.new(self)
+        klass.implement(method)
+        klass
+      end
+
+      def self.implement(method)
+        @name = method
+        @implementation = Math.method(method)
+      end
+
+      def self.name
```

Comment:
I made assumptions regarding which methods were needed or not. Found this one in FunctionRegistry but it was not hit by code coverage, so I'm not sure if it's needed.